### PR TITLE
August Updates

### PR DESCRIPTION
**Pull request by dshanske**
Update dependencies as well as fixes for author_url issues, addition of toggles in the h-card widget, removal of a duplicative global call, etc.

### CODE REVIEW

File: includes/class-hcard-user.php [hunk 1]
```diff
@@ -204,16 +206,16 @@ public static function save_profile( $user_id ) {
...
 		foreach ( $fields as $key => $value ) {
 			if ( isset( $p[ $key ] ) ) {
 				update_user_meta( $user_id, $key, sanitize_text_field( $p[ $key ] ) );
 			} else {
 				delete_user_meta( $user_id, $key );
 			}
 		}
-		if ( isset( $_POST['relme'] ) ) {
-			$relme = explode( "\n", $_POST['relme'] );
+		if ( isset( $_POST['relme'] ) ) { // phpcs:ignore
```
Comment (pfefferle) [inline includes/class-hcard-user.php]: Isn't there a requst/query object that we could use instead?

File: includes/class-hcard-user.php [hunk 2]
```diff
@@ -436,51 +438,53 @@ public static function relme_head_list( $author_id = null ) {
...
+
+	public static function pgp() {
+		$author_id = self::get_author();
+		if ( ! $author_id ) {
 			return;
 		}
 		$pgp = get_user_option( 'pgp', $author_id );
 		if ( ! empty( $pgp ) ) {
-			echo '<link rel="pgpkey" href="' . $pgp . '">';
+			echo '<link rel="pgpkey" href="' . $pgp . '">'; // phpcs:ignore
```
Comment (pfefferle) [inline includes/class-hcard-user.php]: I prefer the XHTML version `<link />`

File: includes/class-hcard-user.php [hunk 3]
```diff
@@ -489,23 +493,29 @@ public static function hcard( $user, $args = array() ) {
...
+		}
+		$url  = $user->has_prop( 'user_url' ) ? $user->get( 'user_url' ) : $url = get_author_posts_url( $user->ID );
+		$name = $user->get( 'display_name' );
 		$email  = $user->get( 'user_email' );
 
+
 		$return  = '<div class="hcard-display h-card vcard p-author">';
 		$return .= '<div class="hcard-header">';
-		$return .= '<a class="u-url url fn" href="' . $url . '" rel="author">';
+		$return .= '<a class="u-url u-uid" href="' . $url . '" rel="author">';
```
Comment (pfefferle) [inline includes/class-hcard-user.php]: no backwards compatbility with mf1?

File: includes/class-relme-widget.php [hunk 4]
```diff
@@ -11,9 +11,9 @@ class RelMe_Widget extends WP_Widget {
 	public function __construct() {
 		parent::__construct(
 			'RelMe_Widget',
-			__( 'Rel=Me', 'indieweb' ),
+			__( 'Rel=Me Links', 'indieweb' ),
```
Comment (pfefferle) [inline includes/class-relme-widget.php]: Isn't the `rel-me` version more common?

**Review by pfefferle — APPROVED** [review]
Looks good to me, but I haven't actively tested the changes...

File: includes/class-hcard-user.php (hunk 1 above)
Comment (dshanske) [reply includes/class-hcard-user.php]: There is $_REQUEST, I suppose, but no function.

Comment (pfefferle) [reply includes/class-hcard-user.php]: no WordPress "native" solution?

Comment (dshanske) [reply includes/class-hcard-user.php]: There probably should be one that doesn't have people touching the $_POST variable, but no. I should suggest that.

File: includes/class-relme-widget.php (hunk 4 above)
Comment (pfefferle) [reply includes/class-relme-widget.php]: Me neither, so you also could keep it like it is... Just asking...

**Comment by pfefferle** [comment]
Can you fix the missing codeclimate issue?

**Comment by dshanske** [comment]
Sorry about that.

**Pull request merged**